2_drivers.yml

```diff
@@ -9,21 +9,44 @@ sources:
 targets:
   2_observations:
     depends:
-      #- out_data/placeholder_for_gefs_archive
-      #- out_data/placeholder_for_nldas
-      #- out_data/reservoir_io.csv
+      - out_data/NLDAS_time[0.377900]_x[397]_y[136].csv
+      - out_data/NLDAS_time[0.377900]_x[401]_y[136].csv
+      - out_data/noaagefs_meteo_driver_files.zip
+      - out_data/reservoir_io_obs.csv
+      - out_data/reservoir_io_sntemp.csv
       
-  # gefs data will come from 
-  #out_data/placeholder_for_gefs_archive:
-   # command: copy_files()
-    
-  #out_data/palceholder_for_nldas:
-   # command: copy_files()
-   
+  # NLDAS Files
+  ## NLDAS_time[0.377900]_x[397]_y[136].csv
+  out_data/NLDAS_time[0.377900]_x[397]_y[136].csv:
+    command: file.copy(from = 'in_data/NLDAS_time[0.377900]_x[397]_y[136].csv', to = target_name)
+
+  ## NLDAS_time[0.377900]_x[401]_y[136].csv
+  out_data/NLDAS_time[0.377900]_x[401]_y[136].csv:
+    command: file.copy(from = 'in_data/NLDAS_time[0.377900]_x[401]_y[136].csv', to = target_name)
+  
+  ## NLDAS to reservoir meteo rds  - Not clear on how to incorporate this
+  log/reservoir_meteo_files.yml:
```

Comment:
curious about how this works when this target is built here.
I ask because it relies on out_data/reservoir_nml_list.rds which is build from out_data/NLDAS_time[0.377900]_x[397]_y[136].csv and out_data/NLDAS_time[0.377900]_x[401]_y[136].csv and that happens in the remake.yml L135

Reply:
These are now directly added in the `remake.yml` file in drivers, no need to add this target.

Reply:
Will remove.